src/main/java/inu/exceptionhandling/ExceptionManager.java

```diff
@@ -0,0 +1,30 @@
+package inu.exceptionhandling;
+
+import inu.task.TaskList;
+
+public class ExceptionManager {
+
+    public static void checkEmptyUserInput(String... strings) throws EmptyUserInputException {
+
+        for (String s : strings) {
+
+            if (s.isEmpty()) {
+
+                throw new EmptyUserInputException();
+
+            }
+        }
+
+    }
```

Comment:
This is a short method, can do without empty lines since it is readable without.